src/custom-element.js

```diff
@@ -192,15 +196,23 @@ function createBaseCustomElementClass(win) {
       const nonStructThis = /** @type {!Object} */ (this);
 
       // `opt_implementationClass` is only used for tests.
+      /** @type {?(typeof ../base-element.BaseElement)} */
       let Ctor =
         win.__AMP_EXTENDED_ELEMENTS &&
         win.__AMP_EXTENDED_ELEMENTS[this.localName];
       if (getMode().test && nonStructThis['implementationClassForTesting']) {
         Ctor = nonStructThis['implementationClassForTesting'];
       }
-      devAssert(Ctor);
-      /** @private {!./base-element.BaseElement} */
-      this.implementation_ = new Ctor(this);
+
+      /** @private {?(typeof ../base-element.BaseElement)} */
+      this.implClass_ = Ctor === ElementStub ? null : Ctor || null;
```

Comment:
This is strange? Can the code that assigns ElementStub to the map assign null instead?

Reply:
I agree. Unfortunately, a couple of attempts to get rid of it failed because a lot of related code treats `null|undefined` the same for this purpose. So, in the meantime, I decided that this is an ok compromise. We can follow up with the complete removal. As it stands - this class is not needed. I'm ok even to do a fast followup PR. But wanted to keep this PR relatively rollback-safe.